operator_ui/src/pages/Chains/ChainConfigFields.tsx

```diff
@@ -0,0 +1,628 @@
+import {
+  Checkbox,
+  FormControlLabel,
+  FormLabel,
+  Grid,
+  MenuItem,
+  TextField,
+  Typography,
+} from '@material-ui/core'
+import React, { useState } from 'react'
+import { capitalize } from '@material-ui/core/utils/helpers'
+
+interface Props {
+  initialValues?: ConfigOverrides
+  onChange: (values: ConfigOverrides, error: string) => void
+}
+
+export interface ConfigOverrides {
+  [attr: string]: JSONPrimitive
+}
+
+type NonNullableJSONPrimitive = string | number | boolean
+
+const defaultKeySpecifics = '{}'
+
+const chainTypes = ['arbitrum', 'exchain', 'optimism', 'xdai']
+const gasEstimatorModes = ['BlockHistory', 'FixedPrice', 'Optimism']
```

Comment:
There is also an `Optimism2` gas estimator.
```suggestion
const gasEstimatorModes = ['BlockHistory', 'FixedPrice', 'Optimism', 'Optimism2']
```